File: app/src/main/java/com/abitcreative/popularmovies/fragments/SettingsFragment.java
package com.abitcreative.popularmovies.fragments;

import android.os.Bundle;
import android.preference.PreferenceFragment;

import androidx.annotation.Nullable;

import com.abitcreative.popularmovies.R;

/**
 * Created by daber on 19/10/16.
 */

public class SettingsFragment extends PreferenceFragment {
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.preferences);
    }
}

File: app/src/main/java/com/abitcreative/popularmovies/fragments/SettingsFragment.kt
package com.abitcreative.popularmovies.fragments

import android.os.Bundle
import android.preference.PreferenceFragment
import com.abitcreative.popularmovies.R

/**
 * Created by daber on 16/10/16.
 */

class SettingsFragment : PreferenceFragment() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        addPreferencesFromResource(R.xml.preferences)
    }
}